docs/system-settings/user-settings.md

```diff
@@ -109,7 +109,9 @@ Preset Palettes           | This is the same as the option in the Patching secti
 ---|------
 Release To Home     | ***On:*** when the last active playback is released fixtures will return to home values.<br>*Off:* LTP channels will remain set.
 Master Release Time | Sets the default release time.
-Release Priority    | The playback priority used with the Release menu or when running Release macros - *Low*, *Normal*, *High*, ***Programmer***, *Very High*.
+Release Priority    | The playback priority used with the Release menu or when running Release macros - *Low*, *Normal*, *High*, ***Programmer***, *Very High*. Playbacks with priority below this level will be released by the *Release All* function (double press \<Release\>).
+
+- You can disable the *Release All* function by setting Release Priority to **Low**. This can be useful to protect against accidentally turning off all playbacks by an unintended double press of \<Release\>.
```

Comment:
I suppose technically this doesn't *disable* it, just makes it so it won't affect most playbacks. Don't know whether this is too nitpicky though :D

Reply:
I have reworded this bit to be clearer